main.go

```diff
@@ -107,6 +107,7 @@ func main() {
 
 		if build.Status != 0 {
 			if strings.TrimSpace(cfg.BuildArtifactsSavePath) != "" {
+				fullBuildArtifactsSavePath := strings.TrimSuffix(cfg.BuildArtifactsSavePath, "/") + "/"
```

Comment:
`cfg.BuildArtifactsSavePath` should be: `strings.TrimSpace(cfg.BuildArtifactsSavePath)`.

This also brings me to the conclusion it worth introducing something like `buildArtifactSaveDir := strings.TrimSpace(cfg.BuildArtifactsSavePath)`